components/brave_extension/extension/brave_extension/content_element_picker.ts

```diff
@@ -264,11 +276,17 @@ const quitElementPicker = () => {
   document.removeEventListener('scroll', elementPickerViewportChanged)
 }
 
-const launchElementPicker = () => {
-  pickerFrame = document.createElement('iframe')
+const attachElementPicker = async () => {
```

Comment:
Hm, looks like after using webpack assets we don't need async anymore. I will remove it.